### PR TITLE
Move toDString, iequals, toCStringThen from dmd.utils to dmd.root.string

### PR DESCRIPTION
**Pull request by ibuclaw**
This is so that dmd.root no longer imports symbols from the toplevel dmd namespace (as well as all the dependencies that come with it from dmd.globals).

### CODE REVIEW

**Review by Geod24 — APPROVED** [review]
Yeah we should have done that a while ago

File: src/dmd/root/response.d [hunk 1]
```diff
@@ -16,7 +16,6 @@ module dmd.root.response;
 
 import dmd.root.file;
 import dmd.root.filename;
-import dmd.utils;
```
Comment (Geod24) [inline src/dmd/root/response.d]: It's used in a `version (unittest)`

Comment (ibuclaw) [reply src/dmd/root/response.d]: Oh. :-)

Comment (ibuclaw) [reply src/dmd/root/response.d]: Fixed (moved the import inside the unittest).

**Pull request merged**